### PR TITLE
Build/push docker image for azure pipelines

### PR DESCRIPTION
**Pull request by muradsater**

### CODE REVIEW

File: Dockerfile-azure-pipelines [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+RUN pip3 install --target=/app -r requirements.txt --no-deps
+
+# Keep the same folder structure for imports
+COPY cognite/transformations_cli/ /app/cognite/transformations_cli/
+
+# Distroless does not work with Azure pipelines
+FROM python:3.7-buster
+COPY --from=builder /app /app
+ENV PYTHONPATH /app
+
```
Comment (emelvarol-cognite) [inline Dockerfile-azure-pipelines:26]: Is the entry point missing or not needed? 🤔 
```
ENTRYPOINT [ "python", "/app/cognite/transformations_cli/__main__.py" ]
```

Comment (emelvarol-cognite) [reply Dockerfile-azure-pipelines:26]: thanks, i'll approve

**Review by emelvarol-cognite — APPROVED** [review]
👍

File: Dockerfile-azure-pipelines (hunk 1 above)
Comment (muradsater) [reply Dockerfile-azure-pipelines:26]: This Dockerfile is very similar to what we have for Azure Pipelines with the function deploy action:
https://github.com/cognitedata/function-action-oidc/blob/master/Dockerfile-azure-pipelines

**Pull request merged**